quickstarts/microsoft.documentdb/cosmosdb-mongodb-autoscale/main.bicep

```
@@ -194,27 +195,19 @@ resource collection2 'Microsoft.DocumentDb/databaseAccounts/mongodbDatabases/col
         {
           key: {
             keys: [
-              'company_id'
-              'company_address'
+              'customer_id'
+              'order_id'
             ]
           }
           options: {
             unique: true
           }
         }
-        {
-          key: {
-            keys: [
-              '_ts'
-            ]
-            options: {
-              expireAfterSeconds: 2629746
-            }
-          }
-        }
       ]
       options: {
```

```suggestion
      ]
    }
    options: {
```

The ADO build is raising a second warning here:

```
D:\a\1\s\quickstarts\microsoft.documentdb\cosmosdb-mongodb-autoscale\main.bicep(207,7) : Warning BCP037: The property "options" is not allowed on objects of type "MongoDBCollectionResourceOrMongoDBCollectionGetPropertiesResource". Permissible properties include "analyticalStorageTtl". If this is an inaccuracy in the documentation, please report it to the Bicep Team. [https://aka.ms/bicep-type-issues]
```

The Bicep warning would be taken care of by moving `options` to be a direct child of `properties` (it is nested under `properties.resource` here). This would match other CosmosDB quickstarts (e.g., [cassandra-autoscale](https://github.com/Azure/azure-quickstart-templates/blob/b78db5d27fd6344656ebdd30f74eee5d22f40dde/quickstarts/microsoft.documentdb/cosmosdb-cassandra-autoscale/main.bicep#L156)).